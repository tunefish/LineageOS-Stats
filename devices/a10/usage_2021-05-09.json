{
  "10.3": 1,
  "16.0": 81,
  "17.1": 1007,
  "18.1": 651,
  "none": 577
}