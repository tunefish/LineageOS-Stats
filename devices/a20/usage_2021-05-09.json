{
  "17.1": 2437,
  "18.1": 352,
  "none": 28
}