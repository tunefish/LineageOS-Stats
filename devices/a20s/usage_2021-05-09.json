{
  "15.1": 1,
  "17.1": 2498,
  "none": 1
}