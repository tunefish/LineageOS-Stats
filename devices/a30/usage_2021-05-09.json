{
  "17.1": 40,
  "18.0": 793,
  "18.1": 541,
  "none": 4
}