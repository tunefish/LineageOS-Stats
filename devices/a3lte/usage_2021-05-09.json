{
  "15.1": 789,
  "16.0": 293,
  "17.1": 1046
}