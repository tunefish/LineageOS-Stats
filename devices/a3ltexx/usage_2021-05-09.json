{
  "14.1": 1295
}