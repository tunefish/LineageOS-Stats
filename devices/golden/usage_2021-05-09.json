{
  "13.0": 1395,
  "14.1": 5009,
  "15.1": 6,
  "17.1": 13,
  "none": 1
}